packages/browser.smash

```diff
@@ -148,6 +148,20 @@
         }
     }
 
+    * Click each {{element}} {
+        const list = element.split(",");
+        const elems = await $$(list);
+        elems.forEach(elem => elem.click());
+    }
+
+    * Drag {{elementFrom}} in {{elementTo}} {
+        await browser.driver.actions({bridge: true}).dragAndDrop(elementFrom, elementTo).perform();
+    }
+
+    * Drag {{elementFrom}} to {{x}}, {{y}} {
+        await browser.driver.actions({bridge: true}).dragAndDrop(dragPosition, {x: parseInt(x), y: parseInt(y)}).perform();
```

Comment:
Variable `dragPosition` doesn't exist. A unit test would catch that :)

Reply:
Sorry iu didn't test this :)